### ISSUE TITLE
[Suggestion] Online RTC on bootup/Wifi connection

### ISSUE THREAD

**Issue by robonxt** [spruceUI/spruceOS#84]
Been enjoying SpruceUI so far! However, due to the non-existing RTC (if there are any internally and not counting the RTC app), the clock is always off. I noticed that OnionOS had a section where they said they can pull the current time from the internet on start of network connection, and was wondering if that would be added in spruceUI for the A30 the near future?

**Comment by Sundownersport** [comment]
I don’t understand. It does have RTC.

you need to set the time

**Comment by robonxt** [comment]
Yes, I know and have used the RTC app. My issue is that it drifts way too much and was wondering if there was a way to either calibrate it, or set the current time from the internet.

**Comment by XanXic** [comment]
Hey, we implemented this based on your suggestion. This will be in the next updated (Currently titled v2.4.0)
